src/TLuaInterpreter.cpp

```diff
@@ -13142,7 +13142,7 @@ int TLuaInterpreter::getTime(lua_State* L)
     QDateTime time = QDateTime::currentDateTime();
     if (return_string) {
         tm = time.toString(format);
-        lua_pushstring(L, tm.toLatin1().data());
+        lua_pushstring(L, tm.toUtf8().constData());
```

Comment:
This might be an isolated case where only ASCII is used anyhow so that the `toLatin1` usage *is* valid.

Reply:
QT 5 uses localized datetime strings, so I don't think this is necessarily ASCII-only.
In fact the QT docs state 
    Note: If localized month and day names are desired, please switch to using `QLocale::system().toString()` as QDateTime methods shall change to use English (C locale) names at Qt 6.
so we might want to switch to that anyway.